client/src/views/TripHomeView.vue

```diff
@@ -73,90 +60,41 @@
-          Fjellvettreglene sjekket! Jeg har også pakket ekstra snacks for alle. 🍙 Kan ikke vente
-          med å se stjernene fra Rustfjellhei! ✨
-        </p>
-      </div>
-    </div>
-    <div class="flex items-start gap-2.5">
-      <AnonymousUserIcon />
-      <div
-        class="flex flex-col w-full my-3 mr-5 leading-1.5 p-4 border-gray-200 bg-gray-100 rounded-e-xl rounded-es-xl"
-      >
-        <div class="flex items-center space-x-2 rtl:space-x-reverse">
-          <span class="text-sm font-semibold text-gray-900">Aoi Inuyama</span>
-          <span class="text-sm font-normal text-gray-500">11:55</span>
-        </div>
-        <p class="text-sm font-normal py-2.5 text-gray-900">
-          Husk å ta med varme klær, folkens! Det blir kjølig om natten. 🧣 Og Nadeshiko, ikke glem
-          teltet denne gangen! 😅
-        </p>
+    <form v-if="isParticipant" @submit.prevent="createAttachment">
+      <div class="flex flex-col items-left">
+        <textarea v-model="attachment.text"
+          class="w-full px-3 py-2 border border-gray-300 rounded-lg shadow-sm focus:ring-blue-500 focus:border-blue-500 sm:text-sm text-gray-900"
+          placeholder="Skriv et innlegg..." />
+        <input type="file" v-on:change="uploadFile" />
+        <button type="submit"
+          class="inline-flex items-center justify-center px-3 py-2 mt-2 text-xs font-medium text-white bg-blue-700 rounded-lg hover:bg-blue-800 focus:ring-4 focus:ring-blue-300">
+          Publiser
+        </button>
       </div>
-    </div>
-    <div class="flex items-start gap-2.5">
-      <img
-        src="https://cdn.myanimelist.net/r/42x62/images/characters/2/337013.webp?s=e53af3011799a7448ad1804a79bade68"
-        class="object-cover rounded-full h-10 w-10 my-3"
-      />
-      <div
-        class="flex flex-col w-full my-3 mr-5 leading-1.5 p-4 border-gray-200 bg-gray-100 rounded-e-xl rounded-es-xl"
-      >
-        <div class="flex items-center space-x-2 rtl:space-x-reverse">
-          <span class="text-sm font-semibold text-gray-900">Chiaki Oogaki</span>
-          <span class="text-sm font-normal text-gray-500">12:00</span>
+    </form>
+    <div v-for="attachment in sortedAttachments" :key="attachment.id">
+      <div class="flex items-start gap-2.5">
+        <img :src="'/avatars/' + attachment.user.avatar" class="object-cover rounded-full h-10 w-10 my-3" />
+        <div
+          class="flex flex-col w-full my-3 mr-5 leading-1.5 p-4 border-gray-200 bg-gray-100 rounded-e-xl rounded-es-xl">
+          <div class="flex items-center space-x-2 rtl:space-x-reverse">
+            <span class="text-sm font-semibold text-gray-900">{{ attachment.user.name }}</span>
+            <span class="text-sm font-normal text-gray-500">{{ no_format_date(attachment.created_at, false, true)
+              }}</span>
+          </div>
+          <p>{{ attachment.text }}</p>
+          <p v-if="attachment.filepath" class="text-sm font-normal pb-2.5 text-gray-900">
```

Comment:
Filepath ser jo ut som alltid er satt, så bør vel sjekke på filename?

```suggestion
          <p v-if="attachment.filename" class="text-sm font-normal pb-2.5 text-gray-900">
```

Reply:
Ah, det var jo en enkel løsning! :D